include/psp2/promoterutil.h

```diff
@@ -19,6 +19,15 @@ typedef struct ScePromoterUtilityLAUpdate {
 	char path[128];    //!< Directory of extracted LA update data.
 } ScePromoterUtilityLAUpdate;
 
+/** Parameters for scePromoterUtilityPromoteImport() */
+typedef struct ScePromoterUtilImportParams{
+	char path[0x80]; //!< Install path usually (ux0:/temp/game) 
+	char titleid[0xC]; //!< Game titleid
+	uint32_t type; //!< Package Type (0x3 on PSM, 0x1 on VITA)
+	uint32_t unk0; //!< Unknown value (seems to be 0x1 on PSM content but 0x00 on Vita contents?)
```

Comment:
I would suggest renaming this field to "flags" or "attribute".

Reply:
Done.